test/mapper.spec.ts

```diff
@@ -10,7 +10,7 @@ import logger from '../src/logger'
 logger.silent = true
 
 describe('Mapper', () => {
-  it('maps input to streams', done => {
```

Comment:
https://stackoverflow.com/questions/45778192/what-is-the-difference-between-it-and-test-in-jest maybe we should just leave "it" for wording purposes?